typing/ctype.ml

```diff
@@ -878,15 +878,21 @@ let update_level env level ty =
       update_level env level true ty
   end
 
-(* Generalize and lower levels of contravariant branches simultaneously *)
+(* Lower level of type variables inside contravariant branches *)
 
-let rec generalize_expansive env var_level visited ty =
+let rec generalize_expansive env var_level visited contra ty =
```

Comment:
Perhaps it would make sense to rename this function (and the one that is exposed in the mli). Something like `lower_contravariant_variables`?

Reply:
Done (renamed to `lower_contravariant`, to avoid being overly specific).